go/samples/http/pgdb/pg_driver.go

```diff
@@ -1,15 +1,19 @@
 package pgdb
 
 import (
+	"database/sql"
 	"log"
 
 	"github.com/google/sqlcommenter/go/core"
 	gosql "github.com/google/sqlcommenter/go/database/sql"
 	_ "github.com/lib/pq"
 )
 
-func ConnectPG(connection string) *gosql.DB {
-	db, err := gosql.Open("postgres", connection, core.CommenterOptions{EnableDBDriver: true, EnableRoute: true, EnableAction: true, EnableFramework: true, EnableTraceparent: true})
+func ConnectPG(connection string) *sql.DB {
+	db, err := gosql.Open("postgres", connection, core.CommenterOptions{
+		Config: core.CommenterConfig{EnableDBDriver: true, EnableRoute: true, EnableAction: true, EnableFramework: true, EnableTraceparent: true, EnableApplication: true},
+		Tags:   core.StaticTags{},
```

Comment:
If you do not need this, can remove this also

Reply:
done